### ISSUE TITLE
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist

### ISSUE THREAD

**Issue by tr4g** [meteor/meteor#11257]
Hello, I'm getting this warning every time I run `meteor run` or `meteor deploy` . I have version 1.11.1. 
```
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist
```

I found other people having this same issue by looking on google, but no answer either in this issues page or Stackoverflow.

**Comment by filipenevola** [comment]
Every time they release a new version this warning is emitted, this is a good first issue as you can fix this problem following this PR https://github.com/meteor/meteor/pull/11167

**Comment by Maxgit3** [comment]
@filipenevola can I assist with this.  If Tr4g prefers to handle the PR that is fine.

**Comment by filipenevola** [comment]
go ahead @Maxgit3

**Comment by denihs** [comment]
Published minifier-css@1.5.4 [PR](https://github.com/meteor/meteor/pull/11394)
